Core/GDCore/Project/CustomObject.cpp

```diff
@@ -22,8 +22,12 @@ std::unique_ptr<gd::Object> CustomObject::Clone() const {
 // This will be used by the GUI to display custom editors (for sprites for
 // instance)
 std::map<gd::String, gd::PropertyDescriptor> CustomObject::GetProperties() const {
-    const auto &properties = eventsBasedObject.GetPropertyDescriptors();
     auto objectProperties = std::map<gd::String, gd::PropertyDescriptor>();
+    if (!project.HasEventsBasedObject(GetType())) {
```

Comment:
Nice! Certainly less performant, but unlikely to be a bottleneck and much safer in case we somehow end up in a situation where customobject continue to exist after their extensions are gone.